docker-compose/config/remote-asset.jsonnet

```diff
@@ -0,0 +1,26 @@
+local common = import 'common.libsonnet';
+{
+  fetcher: {
+    caching: {
+      fetcher: {
+        http: {
+          allowUpdatesForInstances: ['fuse', 'hardlinking'],
+          contentAddressableStorage: common.blobstore.contentAddressableStorage,
+        },
+      },
+    },
+  },
+
+  assetCache: {
+    actionCache: {
+      blobstore: common.blobstore,
+    },
+  },
+  global: common.global,
+  grpcServers: [{
+    listenAddresses: [':8984'],
+    authenticationPolicy: { allow: {} },
+  }],
+  allowUpdatesForInstances: ['foo'],
```

Comment:
Is there a more descriptive name, or just a comment, about what this instance name refers to?